### ISSUE TITLE
srfi/87 is unusable

### ISSUE THREAD

**Issue by shawnw** [racket/srfi#20]
Always gives syntax errors. For example:

```
> (require srfi/87)
> (case 1 ((1 2 3) 'foo) (else 'bar))
string:1:24: my-case: bad syntax (not a datum sequence)
  at: else
  in: (my-case x (else (quote bar)))
 [,bt for context]

```

https://github.com/shawnw/racket-extra-srfi-libs/blob/master/srfi/87x.rkt for a working version if you want to replace the srfi-lib version with it (This version also uses `equal?` instead of `eqv?` for comparisons like the vanilla Racket `case`)

**Comment by mflatt** [comment]
Another possibility is to adjust `srfi/87` to recognize both `else`s, which should preserve backward compatibility while making it work with `racket/base`.

**Comment by samth** [comment]
I implemented @mflatt's suggestion in #21.
